playground/backend/internal/cache/redis/redis_cache_test.go

```diff
@@ -68,7 +68,7 @@ func TestRedisCache_GetValue(t *testing.T) {
 			wantErr: true,
 		},
 		{
-			name: "all success",
+			name: "get existing value",
```

Comment:
Sometimes you have a name with an upper case letter, but sometimes with lower case. Do the same everywhere

Reply:
Done